src/diff-snapshot.js

```diff
@@ -271,10 +271,10 @@ function diffImageToSnapshot(options) {
     result = { added: true };
   } else {
     const receivedSnapshotPath = path.join(receivedDir, `${snapshotIdentifier}${receivedPostfix}.png`);
-    rimraf.sync(receivedSnapshotPath);
+    rimrafSync(receivedSnapshotPath, { glob: true });
```

Comment:
Is `glob: true` required here?

Reply:
The test cases pass without `glob: true` here.
Without it in `integration.spec.js` an error is thrown stating "Illegal characters in path.". 
Since the path came from options then I´m not sure if it can become a problem without setting glob to true so I added it here as well. If you think this assumption is wrong then I have no problem removing it.